loopin.go

```diff
@@ -80,6 +82,28 @@ func newLoopInSwap(globalCtx context.Context, cfg *swapConfig,
 	currentHeight int32, request *LoopInRequest) (*loopInInitResult,
 	error) {
 
+	var err error
+	var routeHints [][]zpay32.HopHint
```

Comment:
Should we set this to `routeHints = request.RouteHints`? So that if we didn't set `Private` but did set `RouteHints` they'll be used? We've already validated that we don't have both set at this point, but seems we're not actually using `RouteHints`?